Corgibytes.Freshli.Cli.Test/ProgramTest.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.IO;
+using Corgibytes.Freshli.Cli.Test.Common;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Corgibytes.Freshli.Cli.Test
+{
+    public class ProgramTest : FreshliTest
+    {
+        public ProgramTest(ITestOutputHelper output) : base(output) {}
+
+        [Fact]
+        public void Validate_Main_loglevel_debug()
+        {
+            StringWriter stringWriter = new StringWriter();
+            Console.SetOut(stringWriter);
```

Comment:
Could some of the repeated code at the start of each test method be moved to a setup function?

Reply:
This has been addressed in [2709615](https://github.com/corgibytes/freshli-cli/pull/46/commits/270961586a28c31c80392046fc7d011334a35e8d).